support/android/fakeld/fake-ld-arm64.sh

```diff
@@ -11,4 +11,5 @@ set -o pipefail
 source ./support/android/fakeld/fake-ld.sh
 
 export _GCC_PARAMS="${@}"
-call_gcc "arch-arm64" "aarch64-linux-android" "android-21" "arm64-v8a"
+call_gcc "arch-arm64" "aarch64-linux-android" "android-21" "arm64-v8a" \
+         "aarch64-linux-android"
```

Comment:
@bors-servo r- 

Actually, `aarch64-linux-android` is duplicated here. Is this intentional?

Reply:
Yes. There are other linker scripts that pass non-identical values to these parameters iirc.

Reply:
@bors-servo r+